_posts/dart-cuts.md

```diff
@@ -45,52 +44,52 @@ and finally
 
 ## Garland & Richardson
-The councils of Garland and Richardson both unanimously voted in support of the current funding levels of DART, a stark difference from their neighbors. Downtown Garland and CityLine in Richardson have been seeing a lot of development effort around their transit centers and it's paying dividends; so maybe it's not too surprising they're against this.
+Thankfully, the councils of Garland and Richardson unanimously voted in support of DART's current funding levels, a stark contrast to their neighbors. Downtown Garland and CityLine in Richardson have been seeing a lot of development efforts around their transit centers, and it's paying dividends - so maybe it's not too surprising they're against these cuts.
 
 ## Carrollton & Farmers Branch
-Carrollton and Farmers Branch had their meetings on the same day, with the same result, but the way they went about it couldn't be any different. 
+July 16th - Carrollton and Farmers Branch had their meetings on the same day, with the same result - but the way they went about it couldn't be any different. 
 
-Carrollton's agenda item was titled "Supporting Transit 2.0". It had a lot of formal text talking about what Transit 2.0 is and why they support it, but if you read all the way to the bottom you see a little blurb saying that in support of Transit 2.0 they support a 25% funding cut to DART. Carrollton has a large TOD being built around their Trinity Mills station, it has a connection to the Denton A-Train, and it's getting the new Silver Line station in Downtown Carrolton. They knew that cutting DART at this time of growth would be unpopular, so they hid it under language that sounds positive for transit.
+Carrollton's agenda item was titled "Supporting Transit 2.0". It had a lot of formal text talking about what Transit 2.0 is and why they support it, but if you read all the way to the bottom, you saw a little blurb saying that in support of Transit 2.0, they support a 25% funding cut to DART. Carrollton has a large Transit-Oriented Development being built around their Trinity Mills station, it has a connection to the Denton A-Train, and it's getting a new Silver Line station in Downtown Carrolton. They knew that cutting DART at this time of growth would be unpopular, so they hid it under language that sounds positive for transit.
```

Comment:
“If you read all the way to the bottom you’ll see”

I think changing it to “you’ll” and putting it back to “see” makes more sense